ee/modules/502-delivery/crds/werfsource.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: werfsources.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: delivery
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: werfsources
+    singular: werfsource
+    kind: WerfSource
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          description: |
+            Delivery source configuration for Argo CD and werf bundle. It is used to create a registry
+            configuration for Argo CD Image Updater and optionally to create a OCI Helm repository
+            in Argo CD to use with werf bundles.
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - imageRepo
+              properties:
+                imageRepo:
+                  type: string
+                  description: Image repository.
+                  x-example: registry.example.com/my-project
+                  pattern: ^(?:[a-z0-9]+(?:[.-][a-z0-9]+)*)(/(?:[a-z0-9._-]+))*$
+                apiURL:
+                  type: string
+                  pattern: ^https?://(?:[a-z0-9]+(?:[.-][a-z0-9]+)*)$
```

Comment:
I wonder whether apiURL can be specified with the http schema?

Reply:
Yes, it is validates. Additionally verified that in cluster.